### PR TITLE
removed padding; added min-width

### PR DESCRIPTION
**Pull request by brianbodollo**
@srph take a peek on this

### CODE REVIEW

File: styles/components/pagination.scss [hunk 1]
```diff
@@ -38,7 +38,7 @@
 
   > .pages > .page {
     display: inline-block;
-    padding: 2px 5px;
```
Comment (srph) [inline styles/components/pagination.scss:41]: It should still have a padding.

**Comment by brianbodollo** [comment]
pagination

how bout this one?

```
min-width: 15px;
padding: 2px 6px;
```

**Comment by srph** [comment]
Looks good. But why `6`? It was `5` last time, right?

**Comment by brianbodollo** [comment]
i found it so thin thats why, well if you want to see.. here is 5px

pagination

**Comment by srph** [comment]
Why not increase the `min-width` instead? :smile:

**Comment by brianbodollo** [comment]
pagination

how bout this one?

**Comment by brianbodollo** [comment]
@srph

File: docs/dist/style.css [hunk 2]
```diff
@@ -4741,6 +4741,7 @@ body {
   display: inline-block; }
 .pagination > .pages > .page {
   display: inline-block;
+  min-width: 17px;
```
Comment (srph) [inline docs/dist/style.css:4744]: Recompile

**Pull request merged**